ccdproc/combiner.py

```diff
@@ -739,7 +738,7 @@ def combine(img_list, output_file=None,
             'func': sigma_clip_func,
             'dev_func': sigma_clip_dev_func}
 
-    # Finally Run the input method on all the subsections of the image
+   # Finally Run the input method on all the subsections of the image
```

Comment:
That seems wrong. I would've expected this to throw an IndentationError... But I think we should stick to 4 space indentation even for comments.

Reply:
Good catch! Haven't figured why my editor sometimes misses things like this....

Reply:
[Turns out my *editor* did catch it...my eyes missed it]